src/extractors/custom/weekly.ascii.jp/index.js

```diff
@@ -0,0 +1,31 @@
+export const WeeklyAsciiJpExtractor = {
+  domain: 'weekly.ascii.jp',
+
+  title: {
+    selectors: ['h1[itemprop]'],
```

Comment:
Minor detail: since we're targeting the `h1` having an `itemprop`, let's make the selector: `h1[itemprop="headline"]`

Reply:
Thank you.
I fixed.